helix-term/src/application.rs

```diff
@@ -15,28 +13,26 @@ use helix_view::{
     tree::Layout,
     Align, Editor,
 };
-use serde_json::json;
-use tui::backend::Backend;
-
 use crate::{
     args::Args,
     commands::apply_workspace_edit,
     compositor::{Compositor, Event},
     config::Config,
+    keymap::Keymap,
     job::Jobs,
-    keymap::Keymaps,
     ui::{self, overlay::overlayed},
 };
-
-use log::{debug, error, warn};
```

Comment:
There are a lot of cosmetic changes like the use declaration reorderings here or fields/variable renamings like `doc`->`description` below. Unless the name/ordering is horribly misleading you should avoid making these changes because it makes the PR harder to review and more likely to incur conflicts with other open PRs

Reply:
I gave my reason for those in the commits. It's easier to following the reasoning to inspect one at the time. There it's clearly shown how "Keymaps" doesn't exist anymore, and that I renamed it to description for it not the be confused with the :help feature that is on it's way.

Reply:
Replacing `keymap::Keymaps`->`keymap::Keymap` in this block is not a problem if the type is changing. I actually didn't notice that was a real change when selecting this block because it's obscured by the unecessary changes to the other use declarations.

d5520e8 should be dropped from this PR: a renaming like that belongs on that feature branch instead and shouldn't be made preemptively

Reply:
Another reasion that I failed to mention explicitly in the commit though, is that im using description for it to stay conistent with the naming of each KeytrieNode. Current code uses doc and desc description for both.
https://github.com/helix-editor/helix/blob/17acadb305bebfef094e4fcc9ba1e6fda6159c3b/helix-term/src/keymap.rs#L82-L99

Reply:
Compare this to:

https://github.com/gibbz00/helix/blob/keymap-presentation/helix-term/src/keymap/keytrie.rs#L107-L130